ci/requirements-py35-min.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
     - pytest
     - pytest-cov
     - pytest-mock
+    - pytest-remotedata
```

Comment:
is the python 3.5 pytest-remotedata package too old? If so delete it here and make the comment below more clear (e.g. "conda package is 0.x, but we need > 0.3.1")